app/templates/application.hbs

```diff
@@ -1,9 +1,11 @@
-<main-navbar></main-navbar>
+<div class="ember-view">
```

Comment:
I think we can safely remove this `div`, can you try?

Reply:
Yep, seems to work 👍